radix-engine/src/transaction/transaction_executor.rs

```diff
@@ -71,6 +72,19 @@ impl Default for CostingParameters {
             archive_storage_price: ARCHIVE_STORAGE_PRICE_IN_XRD.try_into().unwrap(),
         }
     }
+    #[cfg(feature = "coverage")]
+    fn default() -> Self {
+        Self {
+            execution_cost_unit_price: Decimal::zero(),
+            execution_cost_unit_limit: u32::MAX,
+            execution_cost_unit_loan: u32::MAX,
+            finalization_cost_unit_price: Decimal::zero(),
+            finalization_cost_unit_limit: u32::MAX,
+            usd_price: USD_PRICE_IN_XRD.try_into().unwrap(),
+            state_storage_price: Decimal::zero(),
+            archive_storage_price: Decimal::zero(),
+        }
```

Comment:
Due to the difference in configuration, some tests may fail in `coverage` mode but not in regular test mode. I guess it's not a big deal for most cases. We will need to make sure the `--no-fail-fast` flag is provided when running coverage.